recipes/pannb/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "pannb" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pannb-{{ version }}.tar.gz
+  sha256: 7df407f4c15173648dab99a9252c551274c76f24dd9e812687aaa2a60dd65a36
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pannb = pannb:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - poetry >=1,<2
+  run:
+    - panflute >=2.1,<3.0
+    - python >=3.8
+  run_constrained:
+    - pandoc >=2.11.2,<2.15
+    - coloredlogs >=14,<16
+
+test:
+  imports:
+    - pannb
+  commands:
+    - pip check
+    - |
+      echo '# testing' | pandoc -F pannb
+  requires:
+    - pip
+    - pandoc >=2.11.2,<2.15
```

```suggestion
    - pandoc
```

Can you try this to make sure `run_constrained` is properly set up?

---

What do you mean? What's wrong with it?

Cf. https://github.com/conda-forge/pantable-feedstock/blob/master/recipe/meta.yaml#L34

---

There's nothing wrong with it, but you shouldn't need the constrains here it thanks to, well, `run_constrained`.

---

Thanks. It would have been more useful if you accompany your request with reason.

Anyway, I think it is better practice to separate the 2 constraints (although in this case the 2 are the same.) supporting versions and test-against versions might be different. Especially in writing pandoc filters, since the pandoc's output can varies depending on its version, sometimes a test can only pass if pandoc is constrained to a specific version. It's not that other versions are not supported (hence excluded in run constrain) but it has a slightly different output that the filter doesn't care about but will fail a test.